steps/kobiton-app-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,91 @@
+title: Kobiton App Upload
+summary: |
+  Uploads an application to Kobiton App Repo
+description: |
+  This step will take an .apk or .ipa file and upload it to the Kobiton App Repository. You can find your API key at [Kobiton](https://portal.kobiton.com/settings/keys)
+website: https://kobiton.com
+source_code_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo
+support_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo/issues
+published_at: 2021-12-02T21:29:30.928866+07:00
+source:
+  git: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo.git
+  commit: 6ea2b2ee115463bee6398e693210af081628d012
```

Comment:
Please add beyondgrep ack as dependency instead of downloading in the script: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#submodules-and-step-dependencies